[CHORE] [#03] 리액트 환경설정

lacomaco opened the pull request:
웹팩&바벨 설정과
리액트 환경설정과 테스트 라이브러리들 환경설정을 했습니다.

리액트 빌드는 npm run frontEndBuild으로 빌드하실 수 있어요.
리액트만 우선 웹으로 띄우는 기능은 npm run frontEndDev으로 확인하실 수 있어요.

혹시 백엔드와 이름이 충돌할까봐 frontEnd test와 serve 스크립트는 앞에
frontEnd 단어를 붙여서 지었습니다.

테스트 환경은 jest랑 react-testing-tool을 사용해요

src에 있는 __tests__는 환경설정이 제대로 되어서 테스트코드가 잘
작동되는지 확인하기 위해서 넣었어요.

혹시 저만 되는 걸 지도 모르니까

npm run frontEndDev
npm run frontEndTest 한번씩 실행해주세요.

lacomaco committed:
[CHORE] [#03] 리액트 환경설정

웹팩&바벨 설정과
리액트 환경설정과 테스트 라이브러리들 환경설정을 했습니다.

리액트 빌드는 npm run frontEndBuild으로 빌드하실 수 있어요.
혹시 백엔드와 이름이 충돌할까봐 frontEnd test와 실행 스크립트는 앞에
frontEnd 단어를 붙여서 지었습니다.

테스트 환경은 jest랑 react-testing-tool을 사용해요

src에 있는 __tests__는 환경설정이 제대로 되어서 테스트코드가 잘
작동되는지 확인하기 위해서 넣었어요.

혹시 저만 되는 걸 지도 모르니까 한번씩 실행해주세요.

lacomaco committed:
[CHORE] [#03] 리액트 환경설정
빠트린 styled-component를 추가하였습니다.

EarlyHail commented:
```
npm run frontEndDev
npm run frontEndTest
npm run frontEndBuild
```
모두 정상작동 합니다!
:+1::+1::+1:

EarlyHail reviewed (approved):
```
npm run frontEndDev
npm run frontEndTest
npm run frontEndBuild
```
모두 정상작동 합니다!
고생하셨어요!!!
👍👍👍

lacomaco committed:
[REFACTOR] [#03] frontEnd node_modules 파일 이동

frontEnd에서만 사용되는 의존성들은  frontEnd파일에 있는게 맞다고 생각해서
webpack과 babel설정 , package.json도 함께 frontEnd 파일 내부로
들여보냈습니다.

yyjjjj commented:
저도 모두 정상작동 합니다!! 감사합니다 👍 👍 👍

yyjjjj reviewed (approved):
모두 정상작동 합니다! 감사합니다 👍 👍 👍

mon823 commented:
확인이 늦어서 죄송합니다ㅠㅠ

mon823 reviewed (commented):
정상 실행 됩니다
